bug-bounty-list/README.md

```diff
@@ -0,0 +1,57 @@
+# disclose.io/bug-bounty-list
+
+> The community-powered database of ALL known public bug bounty and vulnerability programs, including their safe-harbor status.  
+
+## How to contribute  
+If you've found a new program, new information on an existing program, or think a program should be deleted, you can contribute to bug-bounty-list by issuing a pull request (PR) to the repo. If you're new to Github, [this article](https://www.thinkful.com/learn/github-pull-request-tutorial/#Time-to-Submit-Your-First-PR) is a good primer on how PRs work. 
```

Comment:
```suggestion
If you've found a new program, new information on an existing program, or think a program should be deleted, you can contribute to bug-bounty-list by issuing a pull request (PR) to the repo. If you're new to Github, [this article](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) is a good primer on how PRs work. 
```

Would something like this in GitHub documentation work? We may need a third-party resource if it's too complex, but first-party is probably ideal.

Reply:
It could, yeh. In the meantime my goal is getting a "how-to" out to the community as quickly as we can, but we can definitely extend it in the future.